src/main/java/org/generationcp/breeding/manager/listmanager/util/GermplasmListExporter.java

```diff
@@ -286,12 +286,14 @@ protected GermplasmList getGermplasmListAndListData(final Integer listId) throws
 
 		final Collection<?> columnHeaders = listDataTable.getContainerPropertyIds();
 		final Object[] visibleColumns = listDataTable.getVisibleColumns();
-		final List<String> excludedColumns = this.getExcludedColumn();
+
+		//FGID and MGID are addable columns in BreedingManager but standard in CrossList.
+		//We exclude them so as not to cause conflict in GermplasmExportedWorkbook code handling for these columns
+		final List<String> excludedColumns = new ArrayList<>(Arrays.asList(ColumnLabels.FGID.getName(), ColumnLabels.MGID.getName()));
```

Comment:
My recommendation was to extra as class constant (static) so as to optimize memory usage